tardis/montecarlo/montecarlo.pyx

```diff
@@ -88,15 +64,12 @@ cdef extern from "src/cmontecarlo.h":
         int_type_t reflective_inner_boundary
         int_type_t current_packet_id
 
-    int_type_t montecarlo_one_packet(storage_model_t *storage, rpacket_t *packet, int_type_t virtual_mode)
-    int rpacket_init(rpacket_t *packet, storage_model_t *storage, int packet_index, int virtual_packet_flag)
-    double rpacket_get_nu(rpacket_t *packet)
-    double rpacket_get_energy(rpacket_t *packet)
+    void montecarlo_main_loop(storage_model_t * storage, int_type_t virtual_packet_flag, int nthreads)
     void initialize_random_kit(unsigned long seed)
 
 
 
-def montecarlo_radial1d(model, int_type_t virtual_packet_flag=0):
+def montecarlo_radial1d(model, int_type_t virtual_packet_flag=0, int nthreads=4):
```

Comment:
No, you just have to specify something for a keyword argument. It will take it from yaml and will default to one.